tests/BasicSearchTests.FunctionalTests.Core/AutocompleteFunctionalTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using BasicSearchTests.FunctionalTests.Core.Models;
+using BasicSearchTests.FunctionalTests.Core.TestSupport;
+using Xunit;
+using System.Net;
+
+namespace BasicSearchTests.FunctionalTests.Core
+{
+    public class AutocompleteFunctionalTests
```

Comment:
Oh I meant the unit tests. [For example...](https://github.com/NuGet/NuGetGallery/blob/master/tests/NuGetGallery.Facts/AutocompleteServicePackageVersionsQueryFacts.cs#L13)

If the Gallery functional tests don't have it then that's ok, but I think we should probably it consistent across all our XUnit tests.